config/locales/en.yml

```diff
@@ -1080,7 +1085,8 @@ en:
         editors: >
           This is the list of all persons that are responsible for this
           medium as editor. At least one person has to be given here, and each
-          of these persons gets editing access to this medium
+          of these persons gets editing access to this medium. Only eligible
```

Comment:
"Persons" sounds archaic. Use "people" instead. E.g. refer to Cambridge dictionary:

> Persons (plural) is a very formal word. We only use it in rather legalistic contexts:
> Any person or persons found in possession of illegal substances will be prosecuted.